src/skin/popup.html

```diff
@@ -90,14 +90,17 @@ <h1 id="share_title" class="i18n_share_title overlay_title"></h1>
   </div>
 
 <div id='privacyBadgerHeader'>
-  <a id="options" title="i18n_popup_options_button" href='/skin/options.html' class="control-button grey-button tooltip" style="display:block" target="_blank"><img src="/icons/options.svg" alt="i18n_popup_options_button"></a>
-  <a id="share" title="i18n_popup_share_button" class="control-button grey-button tooltip" style="display:block" target="_blank"><img width="25" height="auto" src="/icons/share.svg" alt="i18n_popup_share_button"></a>
-  <a id="help" title="i18n_popup_help_button" href='/skin/firstRun.html' class="control-button grey-button tooltip" style="display:block" target="_blank"><img src="/icons/help.svg" alt="i18n_popup_help_button"></a>
+  <a id="options" title="i18n_popup_options_button" href='/skin/options.html' class="control-button grey-button tooltip" style="display:block" target="_blank"><img width="25" src="/icons/black-options-icon.svg" alt="i18n_popup_options_button"></a>
+  <a id="share" title="i18n_popup_share_button" class="control-button grey-button tooltip" style="display:block" target="_blank"><img width="25" height="auto" src="/icons/black-share-icon.svg" alt="i18n_popup_share_button"></a>
+  <a id="help" title="i18n_popup_help_button" href='/skin/firstRun.html' class="control-button grey-button tooltip" style="display:block" target="_blank"><img width="25" src="/icons/black-help-icon.svg" alt="i18n_popup_help_button"></a>
   <div id='title'>
-    <img src="/icons/badger-48.png" srcset="/icons/badger-128.png 2x" width="48" alt=""> <h2><span class="i18n_name"></span></h2>
+    <img src="./images/PB-logo-bw.svg" srcset="./images/PB-logo-bw.svg 2x" width="48" alt="privacy badger logo" id="badger-header-logo">
```

Comment:
Do we actually still need the `srcset` property?

Reply:
The `alt` property should stay blank as it's not important to tell screen readers that there is a logo here; the logo is just decorative. We could label the logo "Privacy Badger", but there is no need as we also have actual text that says "Privacy Badger" right next to the logo. Please see https://webaim.org/techniques/alttext/ for more information.

Reply:
I thought the `srcset` attribute was a sort of safety precaution, but I guess since we aren't doing any dynamic rendering of different assets based on screen size, I can take this one and others in this html file out

Reply:
Is there a point to a `srcset` that specifies the same image file twice?

Reply:
I don't think you should remove the other `srcset`s ...

Reply:
Ah, I meant the previous one that I added. It's been removed